### ISSUE TITLE
also needs to observe contentSize property in order to handle pasting many line and deleting many lines in a single action

### ISSUE THREAD

**Issue by eventomer** [sudeep-jaiswal/ASJExpandableTextView#3]
something like this:

```
    [self.KVOController observe:self keyPath:@"contentSize" options:NSKeyValueObservingOptionNew block:^(id observer, id object, NSDictionary *change) {
        typeof(weakSelf) strongSelf = weakSelf;
        if (strongSelf.isExpandable) {
            [strongSelf handleExpansion];
        }
    }];
```

**Comment by sdpjswl** [comment]
will look at this, thanks for raising the issue

**Comment by sdpjswl** [comment]
Fixed in [version 0.4](https://github.com/sudeepjaiswal/ASJExpandableTextView/releases/tag/0.4).
